maturity-model.md

```diff
@@ -9,10 +9,10 @@ Tier | Content
 0 | IDC Minimalization > Project Formation using Socio-Economic Lens & #DifferentTogether Pledge
 1 | Test product/service for minorities: BIPOC, LGBTQIA2S+, Non-Binary, People with Disabilities, Residents in Rural areas
 CI  | Check in - Substantial analysis completed for all groups?
-2 | IDC Acceptance > Intersect product/service analysis with privacy and security
-CI | Check in - 
-3 | Apply Privacy and Security
+2 | IDC Acceptance > Intersect product/service analysis with privacy and security components, network infrastructure and both semantics and syntax of language
+CI | Check in - Where do intersections collide with minority groups? Revist with respective experts if so
+3 | Run language script through inclusive language application
+CI | Is the chosen language absent of us/them concept, anti-oppression, and inclusive for all groups?

```

Comment:
What's going on with CI tiers?

Reply:
A common issue I find is that a bottleneck can be approvals. The CI serves as a prompt to review the work in that lens and that it generates other areas to look into prior to moving to the next step. This reduces the chances of a pending approval because the decision is to 'fix this before coming back'.